packages/earn-controller/src/EarnController.ts

```diff
@@ -503,31 +669,485 @@ export class EarnController extends BaseController<
     address,
   }: RefreshPooledStakingDataOptions = {}): Promise<void> {
     const errors: Error[] = [];
+    for (const chainId of this.#supportedPooledStakingChains) {
+      await Promise.all([
```

Comment:
Instead of using `Promise.all` and adding `catch` handlers to each promise, what are your thoughts on using `Promise.allSettled` instead? I think you should be able to capture the errors in a simpler way.

Reply:
Besides this perhaps we ought to note that `refreshPooledStakingData` now pulls more kinds of data? (feel free to be a bit more specific, I'm not sure what this does)

Reply:
As far as pooled staking here the diff is weird but its the same data just across supported networks, and I will note that.